manifests/kubo.yml

```diff
@@ -117,3 +117,20 @@ variables:
   type: password
 - name: kubelet-password
   type: password
+- name: kubo_ca
```

Comment:
Could you please elaborate on why having separate CA for Kubo is better than having CA certificate in credhub?